### DISCUSSION TITLE
posts/rust-collection-ownership/

### DISCUSSION THREAD

**Discussion by giscus** [d4mr/d4mr.github.io#6] [giscus]
# posts/rust-collection-ownership/

Collections and Ownership in Rust, with Illustrated Examples (and Illustrations)

https://d4mr.github.io/posts/rust-collection-ownership/

**Comment by d4mr** [comment]
Test comment to verify giscus integration

**Reply by d4mr** [reply]
Just made a commit that fixed broken light mode! Comments are working well now :)

**Comment by tusharpandey13** [comment]
This was cool
